src/replication.c

```diff
@@ -1892,7 +1903,8 @@ void readSyncBulkPayload(connection *conn) {
             cancelReplicationHandshake(1);
             return;
         }
-        atomicIncr(server.stat_net_input_bytes, nread);
+        /* Repl bytes from fullresync. */
+        atomicIncr(server.stat_net_repl_input_bytes, nread);
```

Comment:
i realize this only counts the full sync bytes in case we're using disk-based replication.
for diskless we'll need to update the metric in `rdbLoadProgressCallback`